palm/environment.py

```diff
@@ -50,7 +50,7 @@ def run_in_shell(self, cmd: str, env_vars: Optional[dict] = {}):
         deprecation_msg = (
             "DEPRECATION: run_in_shell has been renamed to "
             "`run_in_docker` and will be removed in a future version. "
-            "Please update your commands to use ctx.obj.run_in_docker"
+            "Please update your commands to use environment.run_in_docker"
         )
         click.secho(deprecation_msg, fg="yellow")
```

Comment:
If we go the deprecation route, we should make a `deprecation_message` utility that encapsulates some of this:
- receives a message arg
- prefixes with `DEPRECATION:`
- secho with `fg='yellow'`

Reply:
I tried this, but as it turns out, this deprecation message is never being displayed... I couldn't get it to display using this deprecated `run_in_shell` function anywhere.